modules/compute/gke-node-pool/main.tf

```diff
@@ -350,6 +354,10 @@ resource "google_container_node_pool" "node_pool" {
       condition     = !(var.enable_queued_provisioning == true && var.autoscaling_total_min_nodes != 0)
       error_message = "autoscaling_total_min_nodes should be 0 when enable_queued_provisioning is true."
     }
+    precondition {
+      condition     = !(var.placement_policy.type != null && var.num_node_pools > 1 && local.has_gpu)
+      error_message = "Multiple GKE node pools for GPUs is supported only when placement policy is not set."
```

Comment:
not sure the grammar here is correct. Should it be "Multiple GKE node pools for GPUs *are* supported only when placement policy is not set."  ?

Reply:
that being said, good catch on this precondition!